book/src/move-basics/testing.md

```diff
@@ -49,15 +49,15 @@ $ sui move test
 
 ## Test Fail Cases with `#[expected_failure]`
 
-Tests for fail cases can be marked with `#[expected_failure]`. This attribute placed on a `#[test]`
-function tells the compiler that the test is expected to fail. This is useful when you want to test
+Tests for fail cases can be marked with `#[expected_failure]`. This attribute, when added to a `#[test]`
+function, tells the compiler that the test is expected to fail. This is useful when you want to test
 that a function fails when a certain condition is met.
 
-> This attribute can only be placed on a `#[test]` function.
+> Note: This attribute can only be added to a `#[test]` function.
 
-The attribute can take an argument for abort code, which is the expected abort code when the test
-fails. If the test fails with a different abort code, the test will fail. If the execution did not
-abort, the test will also fail.
+The attribute can take an argument specifying the expected abort code that should be returned if the
+test fails. If the test fails with a different abort code, it will fail. Likewise, if execution does
```

Comment:
```suggestion
test fails. If the test returns an abort code different from the one specified in the argument, it will fail. Likewise, if execution does
```